src/main/scala/com/guidewire/cda/TableReader.scala

```diff
@@ -484,13 +512,22 @@ class TableReader(clientConfig: ClientConfig) {
    * @return DataFrame that is the same as input, but has irrelevant internal columns removed
    */
   private[cda] def dropIrrelevantInternalColumns(dataFrame: DataFrame): DataFrame = {
-    val dropList = dataFrame.columns.filter(colName => (colName.toLowerCase.startsWith("gwcbi___") && !relevantInternalColumns.contains(colName))
-      // Get rid of geospatial columns.
-      || colName.toLowerCase.contains("spatial")
-      // Get rid of textdata columns. Contains XML data that won't insert into the database properly.
-      || colName.toLowerCase.equals("textdata"))
+    val dropList = dataFrame.columns.filter(colName => (colName.toLowerCase.startsWith("gwcbi___") && !relevantInternalColumns.contains(colName)))
     dataFrame.drop(dropList: _*)
+  }
 
+  /** Function that takes a DataFrame corresponding to some table and drops internal columns that
```

Comment:
This documentation is factually incorrect. Copy-paste error?

Reply:
I'll look at it